__tests__/lib/index.test.js

```diff
@@ -203,13 +200,13 @@ describe( 'install-go-binary', () => {
 	} );
 
 	describe( 'getInstallDir()', () => {
-		const fsAccessSpy = jest.spyOn( fs.promises, 'access' );
-		const fsMkdirSpy = jest.spyOn( fs.promises, 'mkdir' );
-		const fsMkdTempSpy = jest.spyOn( fs.promises, 'mkdtemp' );
+		const fsAccessSpy = jest.spyOn( promises, 'access' );
+		const fsMkdirSpy = jest.spyOn( promises, 'mkdir' );
+		const fsMkdTempSpy = jest.spyOn( promises, 'mkdtemp' );
 
 		it( 'should return the package dir when writable', async () => {
 			fsAccessSpy.mockResolvedValue();
-			fsMkdirSpy.mockResolvedValue();
+			fsMkdirSpy.mockResolvedValue( undefined );
```

Comment:
Here and below: the function is expected to return something (in this case, `undefined` is fine). While the lack of the value is technically `undefined`, the linter does not like this.